08_voting/index.tex

```diff
@@ -1 +1,101 @@
+\end{itemize}
+
+In an IIGO session, the voting action is needed for deciding the next President, Speaker, Judge and also for deciding whether or not to adapt a new rule as well as to disable an active one.
+
+\textbf{Elections of IIGO roles:} As per the IIGO specification, the elections of roles will be conducted according to the accountability cycle (see Section XX): The Judge can initiate the transfer-of-power for the Speaker; the President can initiate the transfer-of-power for the Judge; the Speaker can initiate the transfer-of-power for the President. To prevent multiple-role enactment of the same island, the island that has been selected for a role will be removed from the candidate list for other roles in the current turn\footnote{Note that this assumes that the following rule is in place: The President, Speaker and Judge are all different islands.}. Depending on the sequence of the elections called, the candidates for three roles will consist of maximally six, five, and four islands accordingly. The ballots from the Voters will be in the format of preference lists which rank all candidates in the desired order.
+
+\textbf{Changes of rules:} The President has the power to select a rule from the \emph{rule proposal list} to be voted by the Voters. The Speaker will host the voting for this rule and will be responsible for counting the votes in order to declare the result. The votes from the Voters can only be of the following types: \textbf{Aye} (namely agree with the rule change), \textbf{Nay} (namely disagree with the rule change) or \textbf{Abstain}.
+
+
+\section{Voting Methods}
+\label{sec:VotingMethods}
+
+\begin{table}[H]
+\caption{Available voting methods for each voting scenario}\label{table:votingmethod}
+\begin{center}
+\begin{tabular}{ |p{3cm}||p{3cm}|p{3cm}|  }
+ \hline
+ Voting Methods   & Elections of Roles & Changes of Rules   \\
+ \hline
+ Plurality   &     & \checkmark     \\
+ \hline
+ Borda Count &  \checkmark   &      \\
+ \hline
+ Runoff      &  \checkmark    &        \\
+ \hline
+ Instant Runoff    & \checkmark  &     \\
+ \hline
+ Approval  &  \checkmark    &    \\
+ 
+ \hline
+\end{tabular}    
+\end{center}
+\end{table}
+
+Table \ref{table:votingmethod} indicates the available voting method choices for each voting scenario considered in the coursework.
+
+\textbf{Plurality} will select the relative majority choices as the collective decision. In the IIGO session, the Plurality voting method will be used to decide whether a change of rule goes into effect. When the number of valid ballots (i.e. abstention excluded) is odd, the Plurality method can determine the winner without tied results. Otherwise, the solution to tied results is regimented with the assumption that the proposer of the voting in such events will vote for "agree". Thus, the proposal will be approved if the number of "agree" votes is more than the number of "disagree" votes. 
```

Comment:
dangling modifier + reference to IIGO as if it only happens once using "the". Suggested rewrite for second sentence:

The Plurality voting method will be used to decide whether a change of rule goes into effect in an IIGO session.

Reply:
i.e. seems to be incorrectly punctuated. Suggested rewrite: 
(i.e., abstention excluded)

Reply:
better wording, agreed

Reply:
dangling modifier --> what you mean?

Reply:
its the technical name of the grammatical structure when you have a sub-clause hanging on the front of the sentence, and a clause after it

Reply:
changed